contributing.md

```diff
@@ -81,7 +81,7 @@ void features.add(__filebasename, {
 
 ## Requirements
 
-[Node.js](https://nodejs.org/en/download/) version 13 or later is required.
+[Node.js](https://nodejs.org/en/download/) version 15 is required.
```

Comment:
```suggestion
[Node.js](https://nodejs.org/en/download/) version 15 or later is required. <!-- Also update .nvmrc -->
```

Let's add an .nvmrc file, at least it's useful to some. I noticed that configuring `cd` to also execute that file is a common pattern.